To: vim_dev@googlegroups.com
Subject: Patch 7.3.470
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.470
Problem:    Test 62 fails when compiled without GUI and X11.
Solution:   Don't test :drop when it is not supported.
Files:	    src/testdir/test62.in


*** ../vim-7.3.469/src/testdir/test62.in	2012-02-22 19:13:00.000000000 +0100
--- src/testdir/test62.in	2012-03-07 22:46:17.000000000 +0100
***************
*** 49,55 ****
  :tabclose
  :call append(line('$'), test_status)
  :"
! :"
  :" Test for ":tab drop exist-file" to keep current window.
  :sp test1
  :tab drop test1
--- 49,55 ----
  :tabclose
  :call append(line('$'), test_status)
  :"
! :if has('gui') || has('clientserver')
  :" Test for ":tab drop exist-file" to keep current window.
  :sp test1
  :tab drop test1
***************
*** 85,90 ****
--- 85,96 ----
  :tabclose
  :q
  :call append(line('$'), test_status)
+ :else
+ :" :drop not supported
+ :call append(line('$'), 'tab drop 1: pass')
+ :call append(line('$'), 'tab drop 2: pass')
+ :call append(line('$'), 'tab drop 3: pass')
+ :endif
  :"
  :"
  :/^Results/,$w! test.out
*** ../vim-7.3.469/src/version.c	2012-03-07 22:52:30.000000000 +0100
--- src/version.c	2012-03-07 22:53:40.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     470,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
158. You get a tuner card so you can watch TV while surfing.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
